packages/lb-components/docs/annotation.md

```diff
@@ -287,6 +315,9 @@ const App = () => {
       stepList={stepList}
       goBack={goBack}
       getFileData={getFileData}
+      getFileList={getFileList}
```

Comment:
The name `getFileList` is too much like `getFileInfo`, think if there is a better one ?